.github/workflows/dedupe-yarn.yml

```diff
@@ -0,0 +1,46 @@
+name: 👔 Deduplicate yarn.lock
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    paths:
+      - ./yarn.lock
+
+jobs:
+  format:
+    if: github.repository == 'remix-run/remix'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 🛑 Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.11.0
+
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.FORMAT_PAT }}
```

Comment:
We don't need to pass `token` if we set write permissions (like we do in `no-response`) I think 🤔